components/IscnUploadForm.vue

```diff
@@ -190,9 +301,29 @@ export default class UploadForm extends Vue {
   fileRecords: any[] = []
 
   isOpenFileInfoDialog = false
+  isOpenSignDialog = false
+  isOpenWarningSnackbar = false
+  isOpenKeplr = true
   isSizeExceeded = false
 
   uploadSizeLimit: number = UPLOAD_FILESIZE_MAX
+  uploadStatus: string = ''
+  arweaveFeeTargetAddress: string = ''
+  arweaveFee = new BigNumber(0)
+  sentArweaveTransactionHashes = new Map<
+    string, { transactionHash?: string, arweaveId?: string }
+  >()
+
+
+
+  likerId: string = ''
+  likerIdsAddresses: (string | void)[] = []
```

Comment:
in what case would this be `void[]`?

Reply:
`likerIdsAddresses` was originally directly awaiting the result of a Promise.all